NGitLab.Mock/Pipeline.cs

```diff
@@ -18,7 +18,7 @@ public Pipeline(string @ref)
 
         public int Id { get; set; }
 
-        public int ProjectId { get; set; }
+        public int ProjectId => Project.Id;
```

Comment:
If someone creates a `Pipeline` object manually, the `Project` property will be null
```suggestion
        public int ProjectId => Project?.Id;
```